.github/ISSUE_TEMPLATE/source_bug.yml

```
@@ -0,0 +1,101 @@
+name: 🐞 Issue/Bug report
+description: Report a source issue/bug in Tachiyomi
+title: "Source Name: short description"
+labels: [bug]
```

Likewise, the repo would need to have it's label switch to capital B for this, but looks better in the new labels coming soontm.

```suggestion
labels: [Bug]
```